starboard/android/shared/platform_configuration/BUILD.gn

```diff
@@ -23,6 +23,7 @@ config("platform_configuration") {
     "log",
     "mediandk",
   ]
+  cflags = [ "-fsigned-char" ]
```

Comment:
@y4vor this causes issues in parts of the Chromium builds—how necessary is it for android when it won't even be conforming to the starboard abi?